documentation/developer/code/external_config.xml

```diff
@@ -1,3 +1,8 @@
+<!--
+ This file is released under terms of BSD license
+ See LICENSE file for more information
+ @author: not specified
```

Comment:
Same comment about author

Reply:
Required now, to silence automatic check. As [discussed ](https://github.com/claw-project/claw-compiler/pull/604#issuecomment-808755861) you can put in the author manually yourself.